.github/workflows/buildcheck.yml

```diff
@@ -12,12 +12,21 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: 3.8
-
+      - name: Install pyenv
+        run: |
+          # Needed for using exact python version, which is
+          # required in `Pipfile`.
+          curl https://pyenv.run | bash
+          export PATH="$HOME/.pyenv/bin:$PATH"
+          pyenv install 3.8.5
```

Comment:
you may avoid this one since we already use `actions/setup-python` above, see https://github.com/actions/setup-python#specifying-a-python-version

Reply:
also, would be nice to update to latest security release 3.8.12 https://www.python.org/downloads/release/python-3812/ update that in the Pipfile and all, generate a new `Pipfile.lock` (could be in another PR).

Reply:
thought so, too. but then we have to stick to the official versions, which are installable in the distro. With pyenv we're more flexible. But maybe we can remove the setup-python action.

Reply:
with setup-python you don't need the python version to be in the distro, it's pretty much like `pyenv` (not sure if it's used underneath).

Reply:
In case of 3.8.5 it didn't find it and just listed the ones currently released and active as official packages for the distro.

But that's not a hard requirement to meet. Just thought that we'd be more flexible that way. Either way is fine for me.

As for changing to 3.8.12, it's on my list but thought I'd put that in a separate PR.

Reply:
I left pyenv and removed setup-python action.